packages/titanium-input-validator/src/titanium-input-validator.ts

```diff
@@ -0,0 +1,56 @@
+import { css, html, LitElement } from 'lit';
+import { property, customElement, state } from 'lit/decorators.js';
+
+@customElement('titanium-input-validator')
+export class TitaniumInputValidatorElement extends LitElement {
+  @property({ type: String }) validationMessage = '';
```

Comment:
Add JS docs

Reply:
Was going to do this in the follow up PR as noted but I can do it now